openedx/features/course_experience/static/course_experience/js/CourseHome.js

```diff
@@ -96,7 +96,7 @@ export class CourseHome {  // eslint-disable-line import/prefer-default-export
     const $vcDismissToggle = $('.vc-toggle', $vcMessage);
     const logEventProperties = { courseRunKey: this.courseRunKey };
 
-    Logger.log('edx.course.upgrade.hero.displayed', logEventProperties);
+    Logger.log('edx.bi.course.upgrade.hero.displayed', logEventProperties);
```

Comment:
@mulby: Here is an example of the event rename.  Let me know if there is an issue with this.

Reply:
LGTM - @clintonb can you think of any reason why we shouldn't do this?

Reply:
Go for it. I don't know of anything that relies on those events.